deployment/7_upgrade_2_0.js

```diff
@@ -2,49 +2,18 @@ const semver = require("semver");
 const childProcess = require("child_process");
```

Comment:
I guess `7_upgrade_2_0.js` should to be deleted.

Reply:
I wanted us to have a template for upgrade scripts. I can rename it to `7_upgrade_2_x.js` or `7_upgrade_2_3.js`?

Reply:
Fine to keep it as a template. Since it does a module update (instead of a feature bundle update), which should be rare now, we could maybe name it `module_upgrade_template.js`?

Reply:
okay done